src/Utils/CustomOptions.php

```diff
@@ -29,4 +29,12 @@ class CustomOptions
     public const GOOGLE_API_RESOURCEDEFINITION = 1053;
 
     public const GOOGLE_API_FIELDBEHAVIOR_REQUIRED = 2;
+
+    // Protobuf extension numbers from https://github.com/googleapis/googleapis/blob/master/google/cloud/extended_operations.proto.
+    public const GOOGLE_CLOUD_OPERATION_FIELD = 1149;
```

Comment:
It's interesting that we copy these values over instead of generating new `google.protobuf.FieldOptions` and `google.protobuf.MethodOptions` objects, but I see that this approach is simpler and so if the actual message classes aren't needed, it's probably just as good to leave them out.